docs/explanation/generic-rules.md

```diff
@@ -0,0 +1,68 @@
+
+To provide an example to distinguish between `HostDown` and `HostMetricsMissing`, consider how the health of Alertmanager can be monitored in different deployment scenarios below:
+
+- In Cos Lite, metrics from Alertmanager are scraped by Prometheus. A successful scrape results in `up` = 1 and a failed scrape results in `up` = 0. As a result, we can rely on the `HostDown` alert to notify in case of issues with the scrape i.e. when `up` is 0.
+- In COS HA, a collector such as `opentelemetry-collector` scrapes Alertmanager and then remote writes the collected metrics into Mimir. In this scenario, in Mimir, we either have an `up` of 1 or an absent `up` altogether. Here, we need the `HostMetricsMissing` alert to be aware of the health of Alertmanager. Note that it is possible that the scrape of Alertmanager being made by the aggregator is successful and that `up` is missing because the aggregator is failing to remote write what it has scraped.
+
+## `AggregatorHostHealth` alert group
+The `AggregatorHostHealth` alert rule group focuses explicitly on the health of aggregators (AKA remote writers), such as `opentelemetry-collector` and `grafana-agent`. This group contains the `HostMetricsMissing` and the `AggregatorMetricsMissing` alert rules.
+
+### `HostMetricsMissing` alert rule
+As mentioned before, the purpose of this alert is to notify when metrics are not reaching the Prometheus (or Mimir) database, regardless of whether scrape succeeded. The alert expression executes `absent(up{...})` with labels including the aggregator's Juju topology: `juju_model`, `juju_application`, `juju_unit`, etc. However, when it comes to aggregators, this rule indicates whether alerts from a collector itself are reaching the metrics backend.
+
+When you have an aggregator charm (e.g. `opentelemetry-collector` or `grafana-agent`), this alert is duplicated per unit of that aggregator, i.e. identifying if the unit is missing a time series. For example, if you have 2 units of `opentelemetry-collector`, and one is behind a restrictive firewall, you should receive only one firing `HostMetricsMissing` alert.
+
+```{note}
+By default, the severity of this alert is `warning`. However, when this alert is for a subordinate remote-writing machine charm, its severity is increased to `critical`.
+```
+
+### `AggregatorMetricsMissing` alert rule
+Similar to `HostMetricsMissing`, this alert is a generic rule that is applied to aggregators to ensure the existence of their `up` metric. The difference, however, is that `AggregatorMetricsMissing` will **fire when all units of an aggregator are down**. For this reason, the alert expression executes `absent(up{...})` with labels including the aggregator's Juju topology: `juju_model`, `juju_application`, but leaves out `juju_unit`. If you have 2 units of an aggregator and the `up` metric is missing for both, this alert will fire.
```

Comment:
```suggestion
Similar to `HostMetricsMissing`, this alert is applied to aggregators to ensure their `up` metric exists. The difference, however, is that `AggregatorMetricsMissing` **triggers only when *all units* of an aggregator are down**. For this reason, the alert expression executes `absent(up{...})` with labels including the aggregator's Juju topology: `juju_model`, `juju_application`, but leaves out `juju_unit`. If you have 2 units of an aggregator and the `up` metric is missing for both, this alert will fire.
```
- Removed some wordiness
- Made it clearer around the "all units" part. You could also just bold this instead of the whole thing